source/_patterns/20-molecules/medium/medium.scss

```diff
@@ -81,18 +82,41 @@
       padding: 8px 12px;
     }
   }
+  .media-still__image,
```

Comment:
.media-still__image und .image-figure__image gehören eigentlich nicht in dieses file - eigene modifier?